tests/crds/cloudscale.crossplane.io_objectsusers.yaml

```diff
@@ -0,0 +1,314 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
Please put the `tests/crds` to `.gitignore`, there's no value to vendor them here.